README.md

```diff
@@ -26,6 +26,12 @@ The Tailwind CSS config contains theme that align default config with Vega desig
 
 # 💻 Develop
 
+### Set up
+
+Check you have the correct version of Node. You can [install NVM to switch between node versions](https://github.com/nvm-sh/nvm#installing-and-updating). Then `NVM install`.
+Before you build you will need to `yarn install` in the root directory.
+The repositry includes a number of template .env files for different networks. Copy from these to the .env file before `serve` to lauch app with different network.
```

Comment:
```suggestion
The repository includes a number of template .env files for different networks. Copy from these to the .env file before `serve` to lauch app with different network.
```

Reply:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker 

Highly recommend, particularly if you're borderline illiterate like me